internal/providers/pluginfw/resources/library/resource_library_acc_test.go

```diff
@@ -64,23 +55,69 @@ func TestAccLibraryUpdatePluginFramework(t *testing.T) {
 				`,
 		},
 		acceptance.Step{
-			Template: `data "databricks_spark_version" "latest" {
+			Template: commonClusterConfig + `resource "databricks_library" "new_library" {
+				cluster_id = databricks_cluster.this.id
+				pypi {
+					package = "networkx"
+				}
 			}
-			resource "databricks_cluster" "this" {
-				cluster_name = "cluster-{var.STICKY_RANDOM}"
-				spark_version = data.databricks_spark_version.latest.id
-				instance_pool_id = "{env.TEST_INSTANCE_POOL_ID}"
-				autotermination_minutes = 10
-				num_workers = 0
-				spark_conf = {
-					"spark.databricks.cluster.profile" = "singleNode"
-					"spark.master" = "local[*]"
+			`,
+		},
+	)
+}
+
+// Testing the transition from sdkv2 to plugin framework.
+func TestAccLibraryUpdateTransitionFromSdkV2(t *testing.T) {
+	sdkV2FallbackFactory := map[string]func() (tfprotov6.ProviderServer, error){
```

Comment:
Let's move this as a top-level var (private to this package) so it can easily be reused between tests.